managed/services/telemetry/telemetry.go

```diff
@@ -106,18 +106,41 @@ func (s *Service) Run(ctx context.Context) {
 	defer ticker.Stop()
 
 	doSend := func() {
-		report, err := s.prepareReport(ctx)
+		var settings *models.Settings
+		err := s.db.InTransaction(func(tx *reform.TX) error {
+			var e error
+			if settings, e = models.GetSettings(tx); e != nil {
+				return e
+			}
+			return nil
+		})
 		if err != nil {
-			s.l.Debugf("Failed to prepare report: %s.", err)
+			s.l.Debugf("Failed to retrive settings: %s.", err)
+			return
+		}
+		if settings.Telemetry.Disabled {
+			s.l.Info("disabled via settings")
 			return
 		}
 
-		err = s.send(ctx, report)
+		report, err := s.prepareReport(ctx)
 		if err != nil {
-			s.l.Debugf("Telemetry info not sent, due to error: %s.", err)
+			s.l.Debugf("Failed to prepare report: %s.", err)
 			return
 		}
-		s.l.Debug("Telemetry info sent.")
+
+		s.l.Debugf("\nTelemetry captured:\n%s\n", s.Format(report))
```

Comment:
@maxkondr 
<img width="1251" alt="image" src="https://user-images.githubusercontent.com/91831719/180398084-6a98c704-b7e1-4a5e-818c-204577a99a34.png">

Reply:
a bunch of `\n` so that it will be easy copy/paste

Reply:
good